cg/meta/orders/external_data.py

```diff
@@ -149,33 +148,93 @@ def create_dict(self, name: str) -> dict:
         return hk_dict
 
     def configure_housekeeper(self, ticket_id: int, dry_run: bool) -> None:
+        self.set_ticket_params(ticket_id=ticket_id)
         cases = self.status_db.get_cases_from_ticket(ticket_id=ticket_id).all()
-        cust_id = cases[0].customer.internal_id
         for case in cases:
             links = case.links
             for link in links:
                 lims_sample_id = link.sample.internal_id
-                paths: List[str] = self.get_all_paths(
-                    lims_sample_id=lims_sample_id, cust_id=cust_id
-                )
+                paths: List[str] = self.get_all_paths(lims_sample_id=lims_sample_id)
                 if dry_run:
                     LOG.info(
                         "Would have added %s to housekeeper and linked associated files, but this is dry-run",
                         lims_sample_id,
                     )
                     return
                 last_version = self.housekeeper_api.last_version(bundle=lims_sample_id)
+
                 if not last_version:
                     LOG.info("Creating bundle for sample %s in housekeeper", lims_sample_id)
                     hk_dict: dict = self.create_dict(name=lims_sample_id)
                     bundle_result: Tuple[Bundle, Version] = self.create_hk_bundle(
                         bundle_name=lims_sample_id, data_dict=hk_dict, dry_run=dry_run
                     )
                     last_version = bundle_result[1]
-                    self.housekeeper_api.add_commit(bundle_result[0], bundle_result[1])
+                    files = []
```

Comment:
I think files should be declared above the if and be renamed to perhaps "fastq_files_paths"?

Reply:
Moved to new function: `add_files_to_bundles`